manifests/params.pp

```diff
@@ -7,13 +7,20 @@
 #
 class solr::params {
 
-  $jetty_home    = '/usr/share/jetty'
+  if $::lsbdistcodename == 'trusty' {
+    $jetty_home    = '/usr/share/jetty'
+  } else {
+    $jetty_home    = '/usr/share/jetty8'
+  }
   $solr_home     = '/usr/share/solr'
   $solr_version  = '4.7.2'
   $mirror_site   = 'http://www.us.apache.org/dist/lucene/solr'
   $data_dir      = '/var/lib/solr'
   $cores         = ['default']
   $dist_root     = '/tmp'
-
+  if $::lsbdistcodename == 'trusty' {
+    $jetty_package = 'jetty'
+  } else {
+    $jetty_package = 'jetty8'
+  }
```

Comment:
Could this if/then/else block not be merged into the one above ? ie, set both jetty_home and jetty_package at the same time.